src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Person } from '../types/Person';
+
+type DropdownMenuProps = {
+  suggestions: Person[];
+  conditionShowDropdown: boolean;
+  selectPersonFromTheList: (
+    event: React.MouseEvent<HTMLParagraphElement>,
+  ) => void;
+};
+
+const DropdownMenu: React.FC<DropdownMenuProps> = ({
+  suggestions,
+  conditionShowDropdown,
+  selectPersonFromTheList,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      {conditionShowDropdown && (
+        <div className="dropdown-content">
+          <div className="dropdown-item" data-cy="suggestion-item">
+            {suggestions.map(person => (
+              <p
+                className="has-text-link"
+                key={person.slug}
+                onClick={selectPersonFromTheList}
+              >
+                {person.name}
+              </p>
+            ))}
+          </div>
```

Comment:
Let's change this like this to pass the last npm test
```suggestion
            {suggestions.map(person => (
            <div className="dropdown-item" data-cy="suggestion-item">
              <p
                className="has-text-link"
                key={person.slug}
                onClick={selectPersonFromTheList}
              >
                {person.name}
              </p>
              </div>
            ))}
```